src/Tools/ConfigurationSchemaGenerator/ConfigurationSchemaGenerator.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>$(DefaultTargetFramework)</TargetFramework>
+	  <TargetFrameworks>$(AllTargetFrameworks)</TargetFrameworks>
```

Comment:
We shouldn't need to multi-target the generator. It should only need to target `net8.0`.

Reply:
I added multi-targeting so we can have tests for both the STJ 8.0 and 9.0 behavior.
Alternatively, we could only target net8.0, but reference STJ 9.0 from the generator project (and remove the TFM-conditional code). Would you prefer that?

Reply:
> Alternatively, we could only target net8.0, but reference STJ 9.0 from the generator project (and remove the TFM-conditional code). Would you prefer that?

Yes, that is a good idea. I've done that in the latest commit.

Targeting a single TFM (`net8.0`) will help us when we move to ship this tool in a reusable nuget package outside of the repo. #3309.